config/packages/doctrine.yaml

```diff
@@ -31,8 +31,11 @@ doctrine:
                 alias: App
         dql:
             string_functions:
-                JSON_EXTRACT: Bolt\Doctrine\Functions\JsonExtract
-                CAST: DoctrineExtensions\Query\Mysql\Cast
+                JSON_EXTRACT: Bolt\Doctrine\Functions\JsonExtract # Why have a duplicate of the Scienta version?
```

Comment:
I know that it _used_ to be different. I'll check if that's still the case, or if we can fall back to the original. I'll get back to you on that.

Reply:
OK, looked it up: The Scienta versions (like `src/Query/AST/Functions/Mysql/JsonExtract.php`) are specific for SQLite, Mysql or Postgres. Ours extends `AbstractJsonFunctionNode` instead, and can be used on either.